web/src/lib/components/Navbar.svelte

```diff
@@ -119,6 +119,7 @@
                         <li class="block md:hidden md:h-0"><a href={null} on:click|preventDefault={() => goto("/about")} class="modal-button cursor-pointer">About</a></li>
                         <li class="block md:hidden md:h-0"><a href={null} on:click|preventDefault={() => goto("/faq")} class="modal-button cursor-pointer">FAQ</a></li>
                         <li><a href={null} on:click|preventDefault={() => goto("/auctions")}>My auctions</a></li>
+                        <li><a href={null} on:click|preventDefault={() => goto(`/stores/${$user.nym}`)}>My store</a></li>
```

Comment:
This is a bit strange - since the goal of this feature is to have others see your store, not for you to see your own store. But I am not saying it is necessarily wrong to have it here, since we could in the future have the user customize their own store. I guess it's OK.